README.md

```diff
@@ -97,7 +100,8 @@ var dogstatsdConfig = new StatsdConfig
     StatsdPort = 8125,
 };
 
-DogStatsd.Configure(dogstatsdConfig);
+if (!service.Configure(dogstatsdConfig))
```

Comment:
You're using `service` here instead of `DogStatsd` like previously, that's on purpose?

Reply:
good catch!